src/trusted/service_runtime/sel_ldr.c

```diff
@@ -1750,3 +1745,14 @@ void InitializeCage(struct NaClApp *nap, int cage_id) {
   nap->cage_id = cage_id;
 }
 
+void print_desctbl(struct NaClApp *nap)
+{
+  printf("--------------------------");
+  printf("printing desc table for nap w cageid: %d\n", nap->cage_id);
+  printf("ptr array at length %d\n", nap->desc_tbl.num_entries);
+  for (int i = 0; i < nap->desc_tbl.num_entries; i++)
+  {
+    printf("Pointer for %d 0x%" PRIXPTR "\n", i, (uintptr_t)nap->desc_tbl.ptr_array[i]);
+  }
+  printf("---------------------------");
```

Comment:
I believe the first and last printfs should either have a newline at the end or be changed to puts.

Reply:
You're right, this is just a function I created for debug anyways. Should probably be removed for this!